README.md

```diff
@@ -123,8 +123,9 @@ oc adm policy add-cluster-role-to-user self-provisioner-namespaces nonadmin
 The container image includes a few lists proven to be successfully executed with non cluster-admin permissions. In order to specify the tests list that the execution will run, the `TESTS` environmental variable is used as `-e TESTS=<filename>`, where filename is:
 
 * [openshift-conformance-minimal](tests-lists/openshift-conformance-minimal.txt). Based on the openshift/conformance/parallel/minimal tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "openshift/conformance/parallel/minimal" | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
-* [openshift-conformance](tests-lists/openshift-conformance.txt). Based on the openshift/conformance tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "openshift/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
-* [kubernetes-conformance](tests-lists/kubernetes-conformance.txt).  Based on the kubernetes/conformance tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "kubernetes/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
+* [openshift-conformance](tests-lists/openshift-conformance.txt). Based on the openshift/conformance tests suite (`openshift-tests run openshift/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
+* [kubernetes-conformance](tests-lists/kubernetes-conformance.txt).  Based on the kubernetes/conformance tests suite (`openshift-tests run kubernetes/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
```

Comment:
I guess you need to add the `--dry-run` if you only need to extract the list of the tests right?

Reply:
I Agree. Done.